solidus_paypal_braintree.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.email     = 'braintree+gemfile@stembolt.com'
   s.homepage  = 'https://github.com/solidusio/solidus_paypal_braintree'
 
-  s.required_ruby_version = '>= 2.5'
+  s.required_ruby_version = '>= 2.6'
```

Comment:
Is this strictly required? I think this might require us to release a major version because we are breaking compatibility with people using Ruby 2.5.

Reply:
I understand. The reason for the change was because the value doesn't match the rubocop version:
https://github.com/solidusio/solidus_paypal_braintree/blob/a27da1b7d5dc9ea52ef75cec3809dbff6d45b66d/.rubocop.yml#L6

and note, 2.6 is no longer supported by Rubocop. So if we want to avoid a major just to satisfy Rubocop, we'll need to disable rubocop for this line.

What do you prefer @kennyadsl?

Reply:
I'd say to do the same that we do on other extensions. For example, I don't see the target ruby version specified [here](https://github.com/solidusio/solidus_auth_devise/blob/4a4f9f213d45b77cead29f45bf5bfd0d2021ab46/.rubocop.yml) or [here](https://github.com/solidusio/solidus_stripe/blob/master/.rubocop.yml). I'd go with that if possible.